gotham-server/src/storage/db.rs

```diff
@@ -68,6 +73,13 @@ where
             rocksdb_client.put(identifier.as_ref(), v_string.as_ref())?;
             Ok(())
         }
+        DB::Redis(redis_client) => {
+            let identifier = idify(user_id, id, name);
+            let v_string = serde_json::to_string(&v).unwrap();
+            let connect = redis_client.get_connection()?;
+            let _: () = redis::cmd("SET").arg(identifier).arg(v_string).query(&connect)?;
```

Comment:
Extract "SET" in a const to avoid magic strings

Reply:
Also you are explicitly ignoring the return value - What happen if we can't connect?